gms/gossiper.hh

```diff
@@ -103,6 +103,9 @@ private:
     using messaging_service = netw::messaging_service;
     using msg_addr = netw::msg_addr;
 
+    static constexpr std::chrono::minutes apply_timeout{1};
+    static constexpr size_t apply_max_queue_size{10000};
```

Comment:
I don't think we are allowed to lose an update.

Reply:
Ok, let's coalesce them on shard 0 then. @asias how bad is it if shard 0 have a more recent view than the rest of the shards? This can happen if we start coalescing updates on shard 0 and replication to other shards lags behind.

Reply:
We should coalesce in a staging buffer until the last replication completes, then replicate the staging buffer.